pennylane/operation.py

```diff
@@ -542,6 +542,72 @@ def eigvals(self):
         """
         return self._eigvals(*self.parameters)
 
+    @staticmethod
+    def compute_terms(*params, **hyperparams):
+        r"""Static method to define the representation of this operation as a linear combination.
+
+        Each term in the linear combination is a pair of a scalar
+        value :math:`c_i` and an operator :math:`O_i`, so that the sum
+
+        .. math:: O = \sum_i c_i O_i
+
+        constructs this operator :math:`O`.
+
+        A ``NotImplementedError`` is raised if no representation by terms is defined.
+
+        .. note::
+            This method gets overwritten by subclasses to define the linear combination representation
+            of a particular operator.
+
+        Args:
+            params (list): trainable parameters of this operator, as stored in the ``parameters`` attribute
+            hyperparams (dict): non-trainable hyperparameters of this operator, as stored in the
+                ``hyperparameters["ops"]`` attribute
+
+        Returns:
+            list[tensor_like or float], list[.Operation]: list of coefficients and list of operations
+
+        **Example**
+
+        >>> qml.Hamiltonian().compute_terms([1., 2.], [qml.PauliX(0), qml.PauliZ(0)])
+        [1., 2.], [qml.PauliX(0), qml.PauliZ(0)]
```

Comment:
I laughed a bit at how trivial this example looks 😆 Although I suppose there is no other implementation we can use here currently

Reply:
I fear not ;)